libs/langchain/langchain/document_loaders/parsers/pdf.py

```diff
@@ -263,24 +263,94 @@ def lazy_parse(self, blob: Blob) -> Iterator[Document]:
 
 class DocumentIntelligenceParser(BaseBlobParser):
     """Loads a PDF with Azure Document Intelligence
-    (formerly Forms Recognizer) and chunks at character level."""
+    (formerly Forms Recognizer). Returns Document with
+    pages or paragraphs, table headers, and rows."""
 
-    def __init__(self, client: Any, model: str):
+    def __init__(self, client: Any, model: str, split_mode: str):
```

Comment:
could we give this default val, probably "page"? so this isn't a breaking change and behavior doesn't change too much

Reply:
I do default it to “page”

https://github.com/annjawn/langchain/blob/c97141309583fc70d6447b9fd216e4b48f09722e/libs/langchain/langchain/document_loaders/pdf.py#L615

Reply:
Yes we can default "page" here as well @baskaryan